packages/vscode-extension/src/webview/views/LaunchConfigurationView.tsx

```diff
@@ -54,7 +54,7 @@ function LaunchConfigurationView() {
       <AppRootConfiguration
         appRoot={appRoot}
         update={update}
-        applicationRoots={applicationRoots}
+        applicationRoots={applicationRoots.map((root) => root.path)}
```

Comment:
any reason we can't just pass the whole objects here? given its called applcationRoots and we now have a structure that is called ApplicationRoot I'd expect this prop to take array of ApplicationRoot objects and not just the paths

Reply:
We can, I just didn't want to refactor that component.
It's no issue.